docs/docs/20-usage/35-advanced-yaml-tricks.md

```diff
@@ -0,0 +1,62 @@
+# Advanced YAML tricks
```

Comment:
Is there a better term than `tricks`? Tricks sounds like some unofficial way of doing it IMO.

Reply:
"usage" ?

Reply:
Or syntax?

Reply:
just push for one and I'll change it ...

Reply:
```suggestion
# Advanced YAML syntax
```